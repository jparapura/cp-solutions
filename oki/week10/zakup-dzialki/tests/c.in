5 9
1 4 9 11 100
7
9
10
55
7
7
7
55
55
