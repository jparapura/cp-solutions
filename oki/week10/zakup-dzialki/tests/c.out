2
0
1
44
2
2
2
44
44
